src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -12,74 +12,183 @@ import { useSelector } from "hooks/useSelector";
+  const attendances = useMemo(() => {
+    if (!usersInVenue) return {};
+
+    // TODO: is this counting logic correct?
+    return usersInVenue.reduce<Record<string, number>>((acc, value) => {
+      Object.keys(value.lastSeenIn).forEach((lastSeenInKey) => {
+        acc[lastSeenInKey] = (acc[lastSeenInKey] ?? 0) + 1;
+      });
+
+      return acc;
+    }, {});
+  }, [usersInVenue]);
+
+  // TODO: do we need this?
+  // const [showEventSchedule, setShowEventSchedule] = useState(false);
 
   return (
     <>
       <div className="party-venue-container">
+        <AnnouncementMessage message={currentVenue.bannerMessage} />
+
+        {/* TODO: should this still be here on the partymap? */}
+        {/*{usersInCamp && (*/}
+        {/*  <div className="row">*/}
+        {/*    <div className="col">*/}
+        {/*      <UserList*/}
+        {/*        users={usersInCamp}*/}
+        {/*        imageSize={50}*/}
+        {/*        disableSeeAll={false}*/}
+        {/*        isCamp={true}*/}
+        {/*        activity={currentVenue.activity ?? "partying"}*/}
+        {/*      />*/}
+        {/*    </div>*/}
+        {/*  </div>*/}
+        {/*)}*/}
+
+        {/* TODO: should this still be here on the partymap? */}
+        {/*<div className="row">*/}
+        {/*  <div className="col">*/}
+        {/*    <div className="starting-indication">*/}
+        {/*      {venue.description?.text}{" "}*/}
+        {/*      {venue.description?.program_url && (*/}
+        {/*          <a*/}
+        {/*              href={venue.description.program_url}*/}
+        {/*              target="_blank"*/}
+        {/*              rel="noopener noreferrer"*/}
+        {/*          >*/}
+        {/*            Event Program here*/}
+        {/*          </a>*/}
+        {/*      )}*/}
+        {/*    </div>*/}
+        {/*    {venue.start_utc_seconds && (*/}
+        {/*        <CountDown*/}
+        {/*            startUtcSeconds={venue.start_utc_seconds}*/}
+        {/*            textBeforeCountdown="Party begins in"*/}
+        {/*        />*/}
+        {/*    )}*/}
+        {/*  </div>*/}
+        {/*</div>*/}
+
         <Map
-          venue={venue}
+          venue={currentVenue}
+          partygoers={usersInVenue}
           attendances={attendances}
           selectedRoom={selectedRoom}
-          setSelectedRoom={setSelectedRoom}
-          setIsRoomModalOpen={setIsRoomModalOpen}
+          selectRoom={selectRoom}
+          unselectRoom={unselectRoom}
         />
+
+        {/* TODO: should this still be here on the partymap? */}
+        {/*<div className="row">*/}
+        {/*  <div className="col">*/}
+        {/*    <RoomList*/}
+        {/*        rooms={venue.rooms}*/}
+        {/*        attendances={attendances}*/}
+        {/*        setSelectedRoom={setSelectedRoom}*/}
+        {/*        setIsRoomModalOpen={setIsRoomModalOpen}*/}
+        {/*    />*/}
+        {/*  </div>*/}
+        {/*</div>*/}
```

Comment:
Not in the designs, ok to remove it.

Reply:
Update: it would be better to keep `RoomList` in the same as `UserList`, but default it to off. This allows us to turn it on in client demos and see what we can learn about its importance.

It also has an important role to play in serendipitous discovery of rooms one might not have paid attention to otherwise, because the events program is featured on the room list and they're presented in a different order and at equal sizes. So there are several reasons to keep it.

Reply:
backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/134

Reply:
cleaned up in https://github.com/sparkletown/sparkle/pull/840/commits/65c7bb71e136693f9cf501cc03f6b9c434034f87